### PR TITLE
CONCF-73 Value of "ab" param in "data-gpt-page-params"

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/CONCF-73

Second part of this ticket can be found in mercury repo: https://github.com/Wikia/mercury/pull/632

ping @rogatty @hakubo

### CODE REVIEW

File: extensions/wikia/AssetsManager/config.php [hunk 1]
```diff
@@ -237,7 +237,7 @@
 
 $config['abtesting'] = array(
 	'type' => AssetsManager::TYPE_JS,
-	'skin' => [ 'oasis', 'wikiamobile', 'venus' ],
+	'skin' => [ 'oasis', 'wikiamobile', 'venus', 'mercury' ],
```
Comment (hakubo) [inline extensions/wikia/AssetsManager/config.php]: do we need this change even if we hardcode Abtest.js in mercury package?

Comment (dianafa) [inline extensions/wikia/AssetsManager/config.php]: no, we don't

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**